features/supplier/supplier_journey.feature

```diff
@@ -104,3 +104,12 @@ Scenario: Supplier user has 5 failed login attempts and is locked. Login is not
   Given The supplier user 'DM Functional Test Supplier User 3' has 5 failed login attempts
   Then The supplier user 'DM Functional Test Supplier User 3' is 'locked' on the admin Users page
   And The supplier user 'DM Functional Test Supplier User 3' 'can not' login to Digital Marketplace
+
+Scenario: Supplier has forgotten password and requests for a password reset
```

Comment:
This applies to both buyers and suppliers now, but as this is part of the `supplier_journey` steps it's probably fine to leave it as it is.